addons/packages/prometheus/2.27.0/package.yaml

```diff
@@ -137,85 +142,90 @@ spec:
                   description: Additional prometheus rules can be configured in this YAML file.
                   default: null
         alertmanager:
-            type: object
```

Comment:
I'm guessing this was incorrectly indented to the right?

Reply:
I guess so, I don't remember changing the indentation, the IDE might have done that automatically.